cli/atomic_swap/cli.py

```diff
@@ -18,6 +23,8 @@
     print_result,
 )
 
+loop = asyncio.get_event_loop()
```

Comment:
You do not need a loop in the CLI layer.

Reply:
@dmytrostriletskyi, done.